app/src/main/java/org/mozilla/fenix/home/sessioncontrol/SessionControlView.kt

```diff
@@ -58,6 +58,7 @@ internal fun normalModeAdapterItems(
         shouldShowCustomizeHome = true
         items.add(AdapterItem.RecentTabsHeader)
         items.add(AdapterItem.RecentTabItem)
+        items.add(AdapterItem.RecentSyncedTabItem)
```

Comment:
We can probably do a bit better here. We can check for `settings.enableTaskContinuityEnhancements` before adding the item.